### PR TITLE
CSFML: 2.0 -> 2.2

### PR DESCRIPTION
**Pull request by eleweek**
Previous discussion: https://github.com/Homebrew/homebrew/pull/35279

SFML 2.1 is updated to SFML 2.2, so I'm creating a new pull-request with only CSFML changes and closing #35279

### CODE REVIEW

**Comment by asparagui** [comment]
Looks like it's not building on Mountain lion:

Error: problem in /usr/local/Library/Formula/csfml.rb
compile error
/usr/local/Library/Formula/csfml.rb:19: syntax error, unexpected tSTRING_BEG, expecting tAMPER
...make", ".", *std_cmake_args, "-DCMAKE_MODULE_PATH=#{Formula[...
                              ^
==> brew uses csfml							 FAILED
Error: compile error
/usr/local/Library/Formula/csfml.rb:19: syntax error, unexpected tSTRING_BEG, expecting tAMPER
...make", ".", *std_cmake_args, "-DCMAKE_MODULE_PATH=#{Formula[...

Can you see if upstream is still supporting ML?  If not, we can disable.

File: Library/Formula/csfml.rb [hunk 1]
```diff
@@ -16,8 +16,7 @@ class Csfml < Formula
   depends_on "sfml"
 
   def install
-    cp_r "#{Formula["sfml"].share}/SFML/cmake/Modules/", "cmake"
-    system "cmake", ".", *std_cmake_args
+    system "cmake", ".", *std_cmake_args, "-DCMAKE_MODULE_PATH=#{Formula["sfml"].share}/SFML/cmake/Modules/"
```
Comment (mikemcquaid) [inline Library/Formula/csfml.rb]: `*std_cmake_args` needs to be the last argument on Ruby 1.8 (which is used by Mountain Lion and below)

**Comment by eleweek** [comment]
Could anyone please look at the failure? It looks like it is some sort of false positive(timeout): http://bot.brew.sh/job/Homebrew%20Pull%20Requests/23327/version=mountain_lion/testReport/brew-test-bot/mountain_lion/test_csfml/

**Comment by DomT4** [comment]
I've asked the bot to take another look at it once it's finished with its current queue. Might be a little while, there's a bit of a backlog at the moment.

**Comment by eleweek** [comment]
So it now says it can be automatically merged! Is there anything I should do?

**Comment by DomT4** [comment]
Looks good. One of the maintainers will merge it in in the next couple days unless they see any issues.

**Comment by xu-cheng** [comment]
Thanks for the pull request! 🎉 Homebrew depends on contributions from community members like you and we're grateful for your support.

For future reference, please use `CSFML 2.2` as the commit title.

**Pull request closed without merging**